frontend/webadmin/modules/gwt-common/src/main/java/org/ovirt/engine/ui/common/widget/uicommon/storage/DisksAllocationView.java

```diff
@@ -179,11 +178,12 @@ public void edit(DisksAllocationModel model) {
     private void initListeners(final DisksAllocationModel model) {
         model.getPropertyChangedEvent().addListener((ev, sender, args) -> {
             if ("Disks".equals(args.propertyName)) { //$NON-NLS-1$
-                addDiskList(model);
+                // Continue to add disks list
```

Comment:
The issue is that the diskList should be created/refreshed again after changing the property of the quotaType, as it affects the number of the columns (from 5 becoming 4 in case of disabled quota).
Since the model was already created, I triggered addDiskList again, so it's been called for both properties' change.
I can add the addDiskList to both if's but it will gain the same result, doesn't it?

Reply:
yes, it should behave the same - that change makes sense
just in terms of code-perspective we won't have that empty block and if addDiskList won't be called if this listener is ever triggered on a property that is different than `Disks` or `QuotaEnforcmentType`

Reply:
So is it better to add addDiskList to both ifs (a small code duplication)?
I can do that, but just wondering if that's what you mean :)

Reply:
yes, that's exactly what I meant :)